### DISCUSSION TITLE
How to add prefix url globally?

### DISCUSSION THREAD

**Discussion by bannarisoftwares** [meilisearch/product#651] [Feedback & Feature Proposal]
I have loadbalancer, currently we doing path based routing. I want to route the request from "api/search/*" path to ec2 instance. To match the pattern we need to add prefix url globally. So is any way available to add "api/search/" prefix url globally on meilisearch ec2?

**Comment by skapin** [comment]
I'm bumping this question. I have a similar usecase.
We would like to server meilisearch docker under a suffix. like `example.com/<SUFFIX>/...`
All routing provided by meilisearch would need to serve under this suffix/prefix (instead of thinking it's  served on `/`)

We can add strip mecanism, but this lead to inapropriate meilisearch internal builduing (like assets/statics/internal routing)

For example, Bottle server allow `Bottle.mount('/my-prefix/', app)` to do that. 

or `url_prefix` for Flask
```
app = Flask(__name__)
app.register_blueprint(bp, url_prefix='/abc/123')
```

**Comment by macraig** [comment]
Hi @skapin and @bannarisoftwares, thank you for sharing your feedback. This feature is not currently available, and while it's not a priority for us right now, I will keep this discussion open so that others can share their thoughts and upvote if they're interested.

**Comment by skapin** [comment]
We will try to add it in our side. If we manage to do it, we will submit a MR
